apps/docs/src/app/core/component-docs/side-navigation/examples/side-navigation-multiple-selected-example/side-navigation-multiple-selected-example.component.html

```diff
@@ -12,9 +12,12 @@
                 </a>
             </li>
             <li fd-nested-list-item [expanded]="true">
-                <a fd-nested-list-link>
-                    <span fd-nested-list-title>Link 3</span>
-                </a>
+                <div fd-nested-list-content>
```

Comment:
If `fd-nested-list-content` needs to be always used, lets put it inside `fd-nested-list-item` to reduce complexity.

Reply:
It doesn't need to be used always, users can put `fd-nested-list-item` inside, instead of `fd-nested-list-content`.